Integrations/LogRhythmRest/LogRhythmRest.md

```diff
@@ -0,0 +1 @@
+LogRhythm security intelligence
```

Comment:
this needs to contain information of how to properly configure the integration instance, how to generate API key, etc...